### PR TITLE
Copy vrf from google/keytransparency package; we will build on this

### PR DESCRIPTION
**Pull request by rlan35**

### CODE REVIEW

**Comment by mikedoan** [comment]
add a README.md to specify the context and original source of the google code

File: crypto/vrf/p256/p256.go [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+	"crypto/x509"
+	"encoding/binary"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/google/keytransparency/core/crypto/vrf"
+	"github.com/google/trillian/crypto/keys"
+
```
Comment (mikedoan) [inline crypto/vrf/p256/p256.go]: fix import - remove this empty line

**Comment by alajko** [comment]
Do we have to copy code? Can we not `go get` and use?

**Comment by LeoHChen** [comment]
So, this is a hard fork.  Shall we just create a repo and start to commit the code in the repo?  So we can easily update the repo?

**Comment by mikedoan** [comment]
You did not address reviewers' comments?

**Pull request merged**